ontologies/WIP/sierratransformable

```diff
@@ -100,84 +100,136 @@
+   * ignore for now 10. Inaddition to excluding 6,9,4 also exclude 4 and 5 
+   * 11. Populate roles "label" with the contents of subfield e for both 100 and 110.
+   *
+   *
    *
-      * Example:
+   * Example:
    *  "contributors": [
    *     {
    *        "type": "Contributor" , 
    *        "agent": {
    *           "id": "a24s7qf3" ,
-   *           "name": "Jacobs, Philip P. (Philip Peter), 1879-1940.",
+   *           "name": "Jacobs, Philip P.",
+   *           "prefixes": ["Sir"] ,
+   *           "numeration": "II" ,
    *           "type": "Person",
    *           "identifiers": [
    *        {
    *           "value": "2005016805",
    *           "type": "Identifier",
-   *           "identifierScheme": "lcsh"
+   *           "identifierScheme": "lc-names"
    *        }
    *        ]   
-   *     "contributions": [
+   *     "roles": [
    *        {
-   *           "contribution": "editor" ,        
    *           "label": "Editor",
-   *           "type": "Contribution"  
+   *           "type": "ContributionRole"  
+   *        } 
+   *     ] ,
+   *  } ,
+   * ],
+   *
+   * Example:
+   *  "contributors": [
+   *     {
+   *        "type": "Contributor" , 
+   *        "agent": {
+   *           "id": "a24s7qf3" ,
+   *           "name": "International Labour Organisation",
+   *           "type": "Organisation",
+   *           "identifiers": [
+   *        {
+   *           "value": "2005016805",
+   *           "type": "Identifier",
+   *           "identifierScheme": "lc-names"
+   *        }
+   *        ]   
+   *     "roles": [
+   *        {
+   *           "label": "Sponsor",
+   *           "type": "ContributionRole"  
    *        } 
    *     ] ,
    *  } ,
    * ],
    *
    *
-
    *
    */
 
 
    /* Populate wwork:genres. Rules:
    * 
-   * 1. For all bibliographic records use "marcTag": "655"
-   * 2. If more than one subfield is populated then populate platform "label" with concatenated subfields in citation/list order and type as "QualifiedConcept".
-   * 3. If only "a" subfield is populated type as "Concept".   
-   * 4. If 655 is populated with a indentified concept from an authority (as indicated by use of second indicator) then populate id with Wellcome id for concept
-   * 5. populate platform "concept" with subfield "a".
-   * 6. When this concept has associated identifiers populate "identifiers" with value (subfield 0) and identity scheme (second indicator or if indicator 7 look to subfield 2) Secound indicator 0=lcsh 2=mesh
-   * 8. For each subfield, for subfields a-z ingnore 0-7, populate "qualifiers" with a "label", type "Concept" (subfield "y" type as "Period" and type z as "Place") and populate "qualiferType" with the subfield name kebab-case.
+   * 
+   * 1. For all bibliographic records use "marcTag": "655".
+   * 2. If one of subfield v,x,y,z is populated. Populate platform "label" with concatenated subfields v,x,y,z in order and type as "QualifiedConcept".
```

Comment:
Just joined together, no extra punctuation?

Reply:
Originally we had hyphens which I removed because unconfirmed. Will work out what we will use and add today.